tools/clang/test/CodeGenDXIL/hlsl/intrinsics/linang-builins.hlsl

```diff
@@ -0,0 +1,55 @@
+// dxc.exe -T cs_6_9 -E cs_main %s
```

Comment:
This isn't a valid `RUN` which will cause LIT to report a failure since it can't identify the test command to run. There are also no `CHECKS`, which makes me think this isn't supposed to be a test. Can you instead put this source into comments in the `ll` file so that it is tracked as being connected but won't be detected as a test file?

Reply:
Yeah, this was meant as a source file, but I figured I can check for the HLSL intrinisics here before lowering here. Left the HL opcodes here as they are meant to be stable.

Reply:
Fixed the RUN line and there are CHECKs. This is a proper test now